server/src/main/java/org/apache/druid/server/coordinator/DruidCluster.java

```diff
@@ -72,6 +84,11 @@ public Map<String, NavigableSet<ServerHolder>> getHistoricals()
     return historicals;
   }
 
+  public Map<String, NavigableSet<ServerHolder>> getManagedHistoricals()
```

Comment:
Also, the method `getHistoricalsByTier()` should be renamed to `getManagedHistoricalsByTier` and it should use the `managedHistoricals` field rather than `historicals`.